tests/integration/component.go

```diff
@@ -496,4 +510,26 @@ func componentTests(args ...string) {
 			helper.Chdir(originalDir)
 		})
 	})
+
+	Context("Creating component with numeric named context", func() {
+		var context string
+		JustBeforeEach(func() {
+			var err error
+			ts := time.Now().UnixNano()
+			context, err = ioutil.TempDir("", fmt.Sprint(ts))
```

Comment:
Instead of ```fmt.Sprint(ts)``` can you pass something simple like ```1234```.

Reply:
ts is a int64 so cannot directly pass it and We should avoid using constants as the next test which requires similar naming the developer wouldn't remember there is `1234` used somewhere and hence they shouldn't use that ( parallel running test could clash )

Reply:
I initiated the discussion because ```.UnixNano()``` naming looks bit counter intuitive to me. Anyway thanks for the details.

/lgtm